docs/dictionary/command/rename.lcdoc

```diff
@@ -43,7 +43,9 @@ on the same volume.
 > obtaining explicit confirmation from the user.
 
 >*Warning:* If a file with the same file path as the newPath already exists, 
```

Comment:
I suggest:
```
>*Cross-platform note:* On Windows, if a file with the same file path as newPath already
> exists then the rename operation will fail. On other platforms, it the existing file may be
> overwritten. If the operation does fail, then the result will be "can't rename file".
```

Renaming over an existing file may fail on all platforms - so its important to remind that checking the result is important.